### PR TITLE
Fix  bug of 'showall' display issue

### PR DESCRIPTION
**Pull request by wenjzhu**
## Description
Previously we had the issue of data loading happens after clicking "talent" or "capability". However, we expect to see data to display right after loading.

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [x] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/home.html [hunk 1]
```diff
@@ -307,41 +307,56 @@ <h4 class="talent_name"></h4>
...
-                    console.log(items)
-                    all_talents = items
-                    append_talent(all_talents);
-                },
-                function () {
-                    console.log("error occured");
-        });
 
-       filterSelection("all");
+        console.log("show all ")
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/home.html:348]: Could we remove the console message and the commented out code?

Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/home.html]: Hi @wenjzhu, It looks like my earlier comment may not have been clear. I was suggesting deleting unwanted console log messages and as well as to delete commented-out code. Thanks.

Comment (wenjzhu) [reply contributions/catalog/webapps/templates/contribute/home.html]: Sorry I realized I misread your comments before...I will work on cleaning commented code as well as console.

**Comment by sandeep-ps** [comment]
@wenjzhu, Please resolve the merge conflict. Thanks.

File: contributions/catalog/webapps/templates/contribute/home.html [hunk 2]
```diff
@@ -307,41 +305,42 @@ <h4 class="talent_name"></h4>
...
 
     <script>
+        function start(){
+
+            ajax('GET', get_cap_url, null,
+                    function (res) {
+                        var items = JSON.parse(res);
+                        all_capability = items
+                        append_capability(all_capability);
+                        filterSelection('all');
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/home.html:315]: Duplicate function calls in lines 315 and 316. One of these can be removed, right?

Comment (wenjzhu) [reply contributions/catalog/webapps/templates/contribute/home.html:315]: Removed. (sorry I missed this one earlier)

Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/home.html:315]: No issues. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**